src/SmartComponents/Portfolio/Portfolio.js

```diff
@@ -54,18 +88,24 @@ class Portfolio extends Component {
   renderProducts = ({ title, filteredItems, addProductsRoute, editPortfolioRoute, removePortfolioRoute }) => (
     <Fragment>
       <PortfolioFilterToolbar/>
-      { (!this.props.isLoading) &&
+      { !this.props.isLoading && !this.state.isRemoving &&
         <div style={ { marginTop: '15px', marginLeft: '25px', marginRight: '25px' } }>
           <PortfolioActionToolbar title={ title }
             filterItems={ this.filterItems }
             addProductsRoute={ addProductsRoute }
             editPortfolioRoute={ editPortfolioRoute }
             removePortfolioRoute={ removePortfolioRoute }
+            onStartRemovingProducts={ this.startRemovingProducts }
           />
         </div>
       }
       <Route exact path="/portfolio/:id/edit-portfolio" component={ AddPortfolioModal } />
       <Route exact path="/portfolio/:id/remove-portfolio" component={ RemovePortfolioModal } />
+      { this.state.isRemoving &&
```

Comment:
I would probably hook this component to route. So it works the same way how the adding adding products work:
https://github.com/ManageIQ/service_portal-ui/blob/4fb34d37110c0675a55af0883a98719f5411db88/src/SmartComponents/Portfolio/Portfolio.js#L143

It will make the code a bit cleaner